include/yaramod/types/expression.h

```diff
@@ -56,6 +57,7 @@ class Expression
 
 	/// @name Getter methods
 	/// @{
+	uint64_t getUid() const { return _uid; }
```

Comment:
Please make it `std::uint64_t` to make it use C++ namespace and also to align it with the rest of the code base.